qemu/tests/cfg/multi_nics_verify.cfg

```diff
@@ -23,7 +23,6 @@
             Windows:
                 # Windows product limit, it will fail when queues is too large
                 del vcpu_maxcpus
-                smp = 4
```

Comment:
If `del vcpu_maxcpus` and also not set the `smp`, what's the CPU num in your test log?

Reply:
According to the test results, it should be half of the assigned host.

Reply:
No, internally we removed smp, here you also remove vcpu_maxcpus, so the CPU count will equal to socket*core*thread, smp and vcpu_maxcpus will not take effect.

Reply:
@PaulYuuu Please help review it again, thanks a lot.